apps/api/src/modules/stores/stores.service.spec.ts

```diff
@@ -1,20 +1,113 @@
+import { CreateStoreDto, UpdateStoreDto } from '@meal-time/api-interfaces';
 import { ConfigService } from '@nestjs/config';
 import { Test, TestingModule } from '@nestjs/testing';
+import { Stores } from '@prisma/client';
 import { PrismaService } from '../../services/prisma/prisma.service';
 import { StoresService } from './stores.service';
 
 describe('StoresService', () => {
-  let service: StoresService;
+	let service: StoresService;
+	let prisma: PrismaService;
+	let store1: Stores;
 
-  beforeEach(async () => {
-    const module: TestingModule = await Test.createTestingModule({
-      providers: [StoresService, PrismaService, ConfigService],
-    }).compile();
+	beforeEach(async () => {
```

Comment:
As with the e2e file most of the setup probably should be placed in a beforeAll, as it seems unnessesary to run before each test.

Reply:
yes